devito/operations/interpolators.py

```diff
@@ -126,90 +122,96 @@ def __init__(self, sfunction):
     def grid(self):
         return self.sfunction.grid
 
-    @cached_property
+    @property
+    def _weights(self):
+        raise NotImplementedError
+
+    @property
+    def _psym(self):
+        return self.sfunction._point_symbols
+
+    @property
+    def _gdim(self):
```

Comment:
imho, just "dimensions" as we do everywhere else

Reply:
No these are not the dimensions these are the dimensions of the underlying grid which are different than the dimension of the Sparse object and associated interpolator